packages/framework/src/Model/Statistics/StatisticsFacade.php

```diff
@@ -74,4 +77,58 @@ public function getNewOrdersCountByDayInLastTwoWeeks()
 
         return $valueByDateTimeDataPoints;
     }
+
+    /**
+     * @param int $fromDays
+     * @param int|null $toDays
+     * @return int
+     */
+    public function getNewCustomersCount(int $fromDays, ?int $toDays = null)
+    {
+        $startDateTime = new DateTimeImmutable('- ' . $fromDays . ' days midnight');
+        $endDateTime = $this->getToDateTime($toDays);
+
+        return $this->statisticsRepository->getNewCustomersCountBetweenDates($startDateTime, $endDateTime);
+    }
+
+    /**
+     * @param int $fromDays
+     * @param int|null $toDays
+     * @return int
+     */
+    public function getOrdersCount(int $fromDays, ?int $toDays = null)
+    {
+        $startDateTime = new DateTimeImmutable('- ' . $fromDays . ' days midnight');
+        $endDateTime = $this->getToDateTime($toDays);
+
+        return $this->statisticsRepository->getOrdersCountBetweenDates($startDateTime, $endDateTime);
+    }
+
+    /**
+     * @param int $fromDays
+     * @param int|null $toDays
+     * @return int
+     */
+    public function getOrdersValue(int $fromDays, ?int $toDays = null)
+    {
+        $startDateTime = new DateTimeImmutable('- ' . $fromDays . ' days midnight');
+        $endDateTime = $this->getToDateTime($toDays);
+
+        return $this->statisticsRepository->getOrdersValueBetweenDates($startDateTime, $endDateTime);
+    }
+
+    /**
+     * @param int|null $toDays
+     * @return \DateTimeImmutable
+     */
+    protected function getToDateTime(?int $toDays): DateTimeImmutable
```

Comment:
here is return type, why?

Reply:
so i am forced to decide whether to call it with null or int?
why cant i call it without parameter?

Reply:
fixed